pubspec.yaml

```
@@ -40,10 +40,59 @@ dev_dependencies:
+  http: ^1.2.2
 
 flutter:
   uses-material-design: true
   assets:
     - assets/images/
     - assets/sounds/
     - assets/silero_models/
+
+scripts:
+  clean: flutter clean && rm -rf web/node_modules
+  get: 
```

yamllint

[error] trailing spaces

(trailing-spaces)